dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -930,12 +931,12 @@ public void removeContentFromLiveIndex(final Contentlet content) throws DotHiber
         removeContentFromIndex(content, true);
     }
 
-    public void removeContentFromIndexByStructureInode(String structureInode) throws DotDataException {
-        String structureName = CacheLocator.getContentTypeCache().getStructureByInode(structureInode).getVelocityVarName();
-        IndiciesInfo info = APILocator.getIndiciesAPI().loadIndicies();
+    public void removeContentFromIndexByStructureInode(final String structureInode) throws DotDataException {
+        final String structureName = CacheLocator.getContentTypeCache().getStructureByInode(structureInode).getVelocityVarName();
```

Comment:
Deprecated - use ContentTypeAPI.find

Reply:
also, this is open to a possible NPE

Reply:
@erickgonzalez let's deprecate the two getter methods that are not deprecated on that class:
1. getStructureByInode
2. getStructureByVelocityVarName

Reply:
done